ngx_http_upstream_check_module.c

```diff
@@ -241,6 +242,7 @@ struct ngx_http_upstream_check_srv_conf_s {
     ngx_array_t                             *fastcgi_params;
 
     ngx_uint_t                               default_down;
+    ngx_uint_t                               ssl;
```

Comment:
True, but if there was another boolean flag to be added - the we wouldn't have to modify this one :)
Also, `default_down` looks like a boolean flag too.
But anyway, this is a very minor thing and can be skipped :)